docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TimeSeries/DetectChangePointBySsa.cs

```diff
@@ -1,6 +1,8 @@
 using System;
 using System.Collections.Generic;
+using System.IO;
 using Microsoft.ML.Data;
+using Microsoft.ML.Transforms.TimeSeries;
 
 namespace Microsoft.ML.Samples.Dynamic
```

Comment:
>Microsoft.ML [](start = 10, length = 12)

plz remove Microsoft.ML prefix

Reply:
Removed here and everywhere!

---
In reply to: [273623785](https://github.com/dotnet/machinelearning/pull/3213#discussion_r273623785) [](ancestors = 273623785)